types/fbinstant/fbinstant-tests.ts

```diff
@@ -0,0 +1,60 @@
+declare const FBInstant: FBInstant;
+
+export class FBInstantTest {
```

Comment:
Removing this `export` keyword should be OK. You're not actually consuming this API as a module.